src/test/kotlin/uk/gov/gdx/datashare/integration/repository/AcquirerRepositoryTest.kt

```diff
@@ -0,0 +1,30 @@
+package uk.gov.gdx.datashare.integration.repository
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import uk.gov.gdx.datashare.repositories.AcquirerRepository
+import uk.gov.gdx.datashare.uk.gov.gdx.datashare.helpers.builders.AcquirerBuilder
+import uk.gov.gdx.datashare.uk.gov.gdx.datashare.integration.MockIntegrationTestBase
+import java.time.LocalDateTime
+
+class AcquirerRepositoryTest(@Autowired private val acquirerRepository: AcquirerRepository) :
```

Comment:
Out of interest, do we need the autowired attribute?

Reply:
Getting `No ParameterResolver registered for parameter` if I remove it